4.5 release page (#1138)
godotengine/godot-website: collections/_release_4_5/entry-platforms-android-support-for-devices-with-16kb-page-sizes.md

---
type: entry
section: platforms
subsection: android
rank: 2
importance: 4
anchor: support-for-devices-with-16kb-page-sizes
title: Support for devices with 16KB page sizes
text: |

>   A computer has some tricks in its sleeve in order to handle gigabytes of memory. One way it does it is to ["page" memory](https://en.wikipedia.org/wiki/Page_(computer_memory)) in discrete blocks. Paging makes it easier to quickly know where to find data, as the system can quickly jump to the right page to search for a specific address.

Reviewer: ```suggestion
  Computers have a few tricks up their sleeve to handle gigabytes of memory. One such trick is ["paging" memory](https://en.wikipedia.org/wiki/Page_(computer_memory)) in discrete blocks, so that the system can quickly jump to it when looking for a specific address.
```
Not sure if it should be "sleeve" or "sleeves" here.

Reviewer 2: "Sleeves" I reckon.

Author: It should be plural:
https://www.merriam-webster.com/dictionary/have%2Fkeep%20%28something%29%20up%20one%27s%20sleeve

>   Pages can come in multiple sizes, depending on the platform. Since its inception, Android only supported 4KB page sizes, but the Android team [announced](https://developer.android.com/guide/practices/page-sizes) compatibilty with 16KB page sizes from Android 15 onwards. Developers should note though that Google stated, starting on November 1st, 2025, that Google Play will require all new submitted apps targeting Android 15 to support 16KB page sizes.

Reviewer: ```suggestion
  Pages can come in multiple sizes depending on the platform. Since its inception, Android only supported 4KB page sizes, but the Android team [recently announced](https://developer.android.com/guide/practices/page-sizes) compatibility with 16KB page sizes from Android 15 onwards. Developers should note, though, that starting on November 1st, 2025, Google Play will require all new submitted apps targeting Android 15 to support 16KB page sizes.
```

Additionally, does Godot have a standard date format for use in posts/pages/etc? Here the "American" format (Month Day, Year) is used, but I'm not sure if in other places the European/rest-of-the-world format (Day Month Year) is used.

Author: I fixed the highlighted issues, but for the date format, we could go for the day month year format as it is used by the blog posts. (see https://godotengine.org/article/release-candidate-godot-4-5-rc-2/)

>   Fortunately, we got your back. Godot 4.5 is ready and supports this feature.

Reviewer: ```suggestion
  Fortunately, we've got your back; Godot 4.5 supports this feature out of the box.
```
I suppose we need to make sure that "out of the box" fits here, but if so, I think this flows better

---
type: entry
section: platforms
subsection: android
rank: 2
importance: 4
anchor: support-for-devices-with-16kb-page-sizes

> title: Support for devices with 16KB page sizes

Reviewer 3: ```suggestion
title: Support for devices with 16KB pages
```
Same here

text: |
  Computers have a few tricks up their sleeves to handle gigabytes of memory. One such trick is ["paging" memory](https://en.wikipedia.org/wiki/Page_(computer_memory)) in discrete blocks, so that the system can quickly jump to it when looking for a specific address.

>   Pages can come in multiple sizes depending on the platform. Since its inception, Android only supported 4KB page sizes, but the Android team [recently announced](https://developer.android.com/guide/practices/page-sizes) compatibility with 16KB page sizes from Android 15 onwards. Developers should note, though, that starting on 1 November 2025, Google Play will require all new submitted apps targeting Android 15 to support 16KB page sizes.

Reviewer 3: ```suggestion
  Pages can come in multiple sizes depending on the platform. Since its inception, Android only supported 4KB pages, but the Android team [recently announced](https://developer.android.com/guide/practices/page-sizes) compatibility with 16KB pages from Android 15 onwards. Developers should note, though, that starting on 1 November 2025, Google Play will require all new submitted apps targeting Android 15 to support 16KB pages.
```
The pages have specific sizes, unless what is meant is the multiples of a base size but then the wording needs to be changed

Pages are aligned to these sizes, so multiple sizes refers to having options between 4 and 16 I believe

  Fortunately, we’ve got your back; Godot 4.5 supports this feature out of the box.[^support-for-devices-with-16kb-page-sizes-dotnet]

>   [^support-for-devices-with-16kb-page-sizes-dotnet]: **.NET users:** .NET 9 (or higher) is required for 16KB page sizes support.

Reviewer 3: ```suggestion
  [^support-for-devices-with-16kb-page-sizes-dotnet]: **.NET users:** .NET 9 (or higher) is required for 16KB page support.
```
